CRM/Prospect/Setup/CreateProspectWorkflowCaseStatuses.php

```diff
@@ -30,12 +31,36 @@ public function apply() {
    */
   public static function getStatuses() {
     $caseStatuses = [
-      ['name' => 'enquiry', 'label' => 'Enquiry (positive)'],
-      ['name' => 'qualified', 'label' => 'Qualified (positive)'],
-      ['name' => 'in_progress', 'label' => 'In progress (positive)'],
-      ['name' => 'submitted', 'label' => 'Submitted (positive)'],
-      ['name' => 'won', 'label' => 'Won (positive)'],
-      ['name' => 'lost', 'label' => 'Lost (negative)'],
+      [
```

Comment:
@tunbola can we have a new upgrader for this please? otherwise it becomes complicated to manually change. 
Old upgrader is for Adding the Case Status's, so we can add a new one for Adding Status Class to those.

Reply:
@deb1990 , The reason I did this is because the extension work is still in development and is not released yet.
Installing this extension should make available all the functionality out of the box without running any upgraders, what this means is that we need to add a new setup for updating these case statuses so that it can be used on extension installation and in the new upgrader we will add. 
The aim is to have as few setups as possible so that the extension is installed quicker, since we have the advantage of avoiding two separate API calls that can be done in one call, it might be good to go that route.

What I have done is to test that the upgrader will run and do what it is supposed to do. I will apply the changes on dev site and I can give you a simple SQL query to run on your local so there will be no compication

What do you think?

Reply:
Okay, please post the SQL in the ticket itself, to keep it document.